serialization/pytorch.py

```diff
@@ -0,0 +1,126 @@
+"""
+Helpers that are specific for i6_models/PyTorch setups
+"""
+from __future__ import annotations
+
+__all__ = ["build_config_constructor_serializers"]
+
+from collections import OrderedDict
+from dataclasses import fields
+from inspect import isfunction
+from typing import List, Optional, Tuple, TYPE_CHECKING
+
+import torch
+from sisyphus.delayed_ops import DelayedBase
+
+if TYPE_CHECKING:
+    from i6_models.config import ModelConfiguration
+
+from .base import Call, Import
+
+
+def build_config_constructor_serializers(
```

Comment:
If this is imported directly in the package `__init__`, the function name should have `torch` in the name. If this is not imported directly in the package, it's ok without `torch`.

But then, this is not just about PyTorch, but also specifically about i6_models. So I think the function name should make this more clear.
```suggestion
def build_i6_models_config_constructor_serializers(
```